Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -211,105 +216,114 @@ Configure the network that will be enabled for the cluster and services.
     Changing networking configuration after a Ceph cluster has been deployed is NOT
     supported and will result in a non-functioning cluster.
 
-#### Host Networking
+#### Ceph public and cluster networks
 
-To use host networking, set `provider: host`.
+Ceph daemons can operate on up to two distinct networks: public, and cluster. Ceph daemons always
+use the public network, which is the Kubernetes pod network by default.
 
-If the host networking setting is changed in a cluster where mons are already running, the existing mons will
-remain running with the same network settings with which they were created. To complete the conversion
-to or from host networking after you update this setting, you will need to
-[failover the mons](../../Storage-Configuration/Advanced/ceph-mon-health.md#failing-over-a-monitor)
-in order to have mons on the desired network configuration.
+The public network is used for client communications with the Ceph cluster (reads/writes).
 
-#### Multus
-
-Rook supports addition of public and cluster network for ceph using Multus
-
-The selector keys are required to be `public` and `cluster` where each represent:
+If specified, the cluster network is used for internal Ceph replication traffic. This includes
+additional copies of data to be replicated between OSDs during client reads/writes. This also
+includes OSD data recovery(re-replication) when OSDs or nodes go offline. If the cluster network is
+unspecified, the public network is used for this traffic instead.
 
-* `public`: client communications with the cluster (reads/writes)
-* `cluster`: internal Ceph replication network
+Some network configurations allow manually specifying the public and network interfaces that Ceph
+will use for data traffic. Use `addressRanges` to specify this. For example:
 
```

Comment:
For my understanding,

So users will create NAD, but they will not mention the IP's range during NAD configuration but they applying ranges laters after creation during Ceph Cluster creation?

Reply:
and users can apply multple IP' range for cluster or public network?

Reply:
No, the NAD configuration hasn't changed. Users still need to configure the NAD however necessary to connect K8s pods to their network per their needs. 

Independently, users may use the `public` and `cluster` `addressRanges` to fill in the `public_network` and `cluster_network` fields of the Ceph config. Users would do this if:
1. Rook cannot auto-detect the network CIDRs from the NAD (or detects the wrong CIDRs) for some reason
2. They do not want Rook to attempt auto-detection of CIDRs from NADs
3. They are using `network.provider: host` and wish to select host interfaces for Ceph